browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchOptions.java

```diff
@@ -0,0 +1,50 @@
+package com.braintreepayments.browserswitch;
+
+import android.content.Intent;
+import android.net.Uri;
+
+import org.json.JSONObject;
+
+public class BrowserSwitchOptions {
```

Comment:
Need docstrings on this entire class

Reply:
Woops - I meant the entire file needs docstrings, as well!

Reply:
Done!